.github/workflows/notify_repositories.yaml

```
@@ -0,0 +1,39 @@
+name: Notify repositories of changes in definitions
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "definitions/**"
+
+jobs:
+  notify-repositories:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        repository: ${{ fromJson(vars.NOTIFY_REPOS) }}
+
+    steps:
+      - name: Send repository_dispatch to ${{ matrix.repository }}
+        env:
+          GITHUB_TOKEN: ${{ secrets.GH_ACCESS_TOKEN }}
+        run: |
+          TAG="${{ github.event.release.tag_name }}"
+          REPO="${{ matrix.repository }}"
+          echo "Notifying repository: $REPO"
+          curl --version
+          STATUS_CODE=$(curl -s -w "%{http_code}" -o response.txt -X POST \
+            -H "Accept: application/vnd.github+json" \
```

The [GitHub REST API documentation](https://docs.github.com/en/rest/guides/getting-started-with-the-rest-api#headers) requires `Content-Type: application/json` when sending JSON payloads. Add the missing header for proper API interaction.

Reference: [REST API Headers](https://docs.github.com/en/rest/overview/media-types)
```suggestion
            -H "Accept: application/vnd.github+json" \
            -H "Authorization: Bearer ${GITHUB_TOKEN}" \
            -H "Content-Type: application/json" \
```